components/molecules/DSHeader.js

```diff
@@ -10,20 +10,7 @@ export default function DSHeader(props) {
       id="header"
       lang={props.locale}
       linkPath={props.langToggleLink}
-      breadCrumbItems={[
-        {
-          link: t.url_canada_ca,
-          text: t.canada_ca,
-        },
-        {
-          link: t.url_serviceCanada,
-          text: t.serviceCanada,
-        },
-        {
-          link: t.url_myBenefitsAndServices,
-          text: t.myBenefitsAndServices,
-        },
-      ]}
+      breadCrumbItems={props.breadCrumbItems}
```

Comment:
Since the first 2 links are to be on every page, maybe this component should just accept additional breadcrumb trail links. This would also help reduce duplicate code (for example sake, say we get asked to add a crumb link to SC Labs, then we only need to add that here once).